src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/CompletionHandler.cs

```diff
@@ -21,6 +22,21 @@ internal class CompletionHandler : IRequestHandler<CompletionParams, SumType<Com
         private static readonly IReadOnlyList<string> HtmlTriggerCharacters = new[] { "<", "&", "\\", "/", "'", "\"", "=", ":", " " };
         private static readonly IReadOnlyList<string> AllTriggerCharacters = CSharpTriggerCharacters.Concat(HtmlTriggerCharacters).ToArray();
 
+        private static readonly IReadOnlyCollection<string> Keywords = new string[] {
+            "for", "foreach", "while", "switch", "lock",
+            "case", "if", "try", "do", "using"
+        };
+
+        private static readonly IReadOnlyCollection<CompletionItem> _keywordCompletionItems = _keywords.Select(k => new CompletionItem
```

Comment:
```suggestion
        private static readonly IReadOnlyCollection<CompletionItem> _keywordCompletionItems = Keywords.Select(k => new CompletionItem
```
To fix the CI:
https://github.com/dotnet/aspnetcore-tooling/pull/2325/checks?check_run_id=955743253